.editorconfig

```diff
@@ -0,0 +1,22 @@
+# This file is for unifying the coding style for different editors and IDEs
```

Comment:
The idea is that this could be used in editors like Sublime Text. But it is not essential and we can remove if needed.